projects/common/vcu118/vcu118_system_bd.tcl

```diff
@@ -120,6 +127,14 @@ ad_ip_parameter axi_gpio CONFIG.C_GPIO_WIDTH 32
 ad_ip_parameter axi_gpio CONFIG.C_GPIO2_WIDTH 32
 ad_ip_parameter axi_gpio CONFIG.C_INTERRUPT_PRESENT 1
 
+ad_ip_instance util_ds_buf sys_cpu_clk_BUFGCE
+ad_ip_parameter sys_cpu_clk_BUFGCE CONFIG.C_BUF_TYPE {BUFGCE_DIV}
+ad_ip_parameter sys_cpu_clk_BUFGCE CONFIG.C_BUFGCE_DIV 2
+ad_connect sys_cpu_clk_BUFGCE/BUFGCE_CE VCC
+ad_connect sys_cpu_clk_BUFGCE/BUFGCE_CLR GND
+ad_connect sys_cpu_clk_BUFGCE/BUFGCE_I sys_cpu_clk
+ad_connect sys_50m_clk sys_cpu_clk_BUFGCE/BUFGCE_O
+
```

Comment:
Can you please move this code to the quad Mxfe  system_bd.tcl file ?  
It is project specific so I would not put it in the common base design.

Other that that looks good. Passes timing with most QuadMxFE configurations.

Reply:
I moved this code in this PR: https://github.com/analogdevicesinc/hdl/pull/943